content/docs/preferences.md

```diff
@@ -467,3 +450,66 @@ If you prefer to edit XML instead of using the GUI to modify shortcuts, you may
 ### Common Shortcut Mapper Problems
 
 With the introduction of the message area, it is easy to see when a conflict exists between shortcuts.  All you have to do is pick the entry that you _don't_ want to use the conflicted shortcut, and either Clear or Modify the shortcut so there is no longer a conflict.
+
+## Other Toggles and Settings
+
+Not all preferences are handled in dialogs, and those toggles and settings are described here.
+
+### Other Settings-menu Entries
+
+* **Settings > Import > Import Plugins...** will ask you to navigate to a plugin DLL anywhere on your filesystem; it will then copy that DLL into the appropriate folder so that the next time you run Notepad++, the application will load that DLL as a plugin (assuming it is a valid plugin DLL).  This is an alternate way to install a single-DLL plugin, rather than the [other installation methods](../plugins/#how-to-install-a-plugin) of using Plugins Admin or manually putting the DLL and associated files in the right location.  (This menu entry will not work on its own if the plugin requires more files than just its DLL.)
+
+* **Settings > Import > Import Style Themes...** will ask you to navigate to a style theme XML file, and will put it in the appropriate folder so that the next time you run Notepad++, the application will include it when you use the [Style Configurator](../preferences/#style-configurator), the new theme will be listed in the "Select theme:" pulldown.
+
+* **Settings > Edit popup ContextMenu** will open the [`contextMenu.xml` configuration file](config-files/#the-context-menu-contextmenu-xml), so that you can edit your context menu, following the advice for [editing configuration files](../config-files/#editing-configuration-files).
+
+### Preference Toggles in Other Dialogs
+
+* The **Save Session** dialog will have a `☐ Save Folder as Workspace` checkbox, which will be active (able to be toggled) when you have a **Folder as Workspace** opened.  If the checkbox is checked, the current **Folder as Workspace** will be included in the saved session; otherwise, the session will not include any **Folder as Workspace** information.  (Requires Notepad++ v7.9.3 or newer.)
+
+* When saving a new file or otherwise running the **Save As** dialog, there is an `☐ Append extension` checkbox.  If checked, Notepad++ will automatically append the first extension listed in the selected **Save as type**; otherwise, the extension must be manually included in the **File name** field.  (This toggle is new to v7.9.6, and replaces the old MISC preference for setting the save dialog file extension to \*.\* that was present in prior versions.)
```

Comment:
Maybe, it makes sense to list the exact text that was used for the preference so that it is easier to find.
This is from v.7.9.1:
![image](https://user-images.githubusercontent.com/9664141/118806504-d6f37880-b8af-11eb-8245-dcc1bcac53c3.png)